src/lib/common/statistics.cpp

```diff
@@ -192,7 +193,7 @@ void statisticsUpdate(RequestType request, Format inFormat)
   case RtUnsubscribeContextResponse:                     ++noOfRtUnsubscribeContextResponse; break;
   case RtSubscribeResponse:                              ++noOfRtSubscribeResponse; break;
   case RtSubscribeError:                                 ++noOfRtSubscribeError; break;
-
+  case RtContextElementResponse:                         ++noOfContextElementResponse; break;
```

Comment:
I only needed this constant for a call to ContextElementResponse::render.
As a side-effect it is added as a counter in statistics.
However, as I haven't included it in the response to a statistics request, an external user will notice no difference.
No big deal really. What we should do is to look into the statistics some day soon ...